music/src/piano_genie/model_test.ts

```diff
@@ -64,17 +64,17 @@ test('Piano Genie Model Correctness', async (t: test.Test) => {
     // Ascending pattern with sampling.
     for (let i = 0; i < 8; ++i) {
       genie.overrideDeltaTime(0.1);
-      keys.push(genie.next(i, 1., 1337));
+      keys.push(genie.next(i, NOCHORD, 1., 1337));
     }
     // Descending pattern with argmax.
     for (let i = 7; i >= 0; --i) {
       genie.overrideDeltaTime(0.1);
-      keys.push(genie.next(i, 0.));
+      keys.push(genie.next(i, NOCHORD, 0.));
     }
     // Fast trill with temperature 0.5.
     for (let i = 0; i < 8; ++i) {
       genie.overrideDeltaTime(0.05);
-      keys.push(genie.next(3 + (i % 2), 0.5, 1337));
+      keys.push(genie.next(3 + (i % 2), NOCHORD, 0.5, 1337));
```

Comment:
i think NOCHORD should be the default, and it should be the _last_ argument, so that this isn't a breaking change

Reply:
(so that you only have to use it if you don't want the default)